client/hosting/server-settings/components/defensive-mode-card/style.scss

```diff
@@ -0,0 +1,32 @@
+.defensive-mode-card__edge-cache-required {
+	color: var(--color-text-subtle);
+	font-style: italic;
+}
+
+.defensive-mode-card__enabled-description {
+	align-items: center;
+	display: flex;
+	gap: 10px;
+
+	p {
+		margin: 0;
+	}
+}
+
+.defensive-mode-card__enabled-indicator {
```

Comment:
This indicator should pulse. See the [Figma component](https://www.figma.com/design/kvZMElckk83g77JSRWOdU8/Server-Settings-Enhancements-i1?node-id=1907-8155&m=dev) for states.

If we set a transparent border to begin with, we should be able to animate the border color to achieve the desired effect.

Reply:
Ah, thanks. I'll look into it!

Reply:
Let me know if you need any help with the animation.

Reply:
Let’s slow the animation to `1.5s`, but other than that it looks great. Thanks @fredrikekelund!